tox.ini

```
@@ -8,6 +8,15 @@ usedevelop = True
 extras =
 	testing
 
+# Check whether there's dependencies present to even make the script run.
+[testenv:check_depends]
```

Personal opinion: I would give this a more generic name because it might be useful for more than just checking dependencies at some point.

```suggestion
[testenv:script]
```

or it could be some other name, but the idea is that this environment just runs the script, so we name it after that, rather than naming it after whatever assorted things we expect it to check.